src/modules/web-actions/index.js

```diff
@@ -0,0 +1,51 @@
+import {NxusModule} from 'nxus-core'
+import {templater} from 'nxus-templater'
+import _ from 'underscore'
+
+class WebActions extends NxusModule {
+  constructor(opts) {
+    super(opts)
+    this._actions = {}
+    templater.templateDir(__dirname+"/templates/*.ejs")
+    templater.on('renderContext.actions-buttons', ::this._actionsContext)
+    templater.on('renderContext.actions-icons', ::this._actionsContext)
+    
+  }
+  
+  /*
+   * Register an action for a template
+   * @param {String} template
+   * @param {String} label
+   * @param {String} link
+   * @param {Object} options (icon, css, group)
+   */
+  add(template, label, link, options) {
```

Comment:
I think ideally an action should be able to provide its own partial which is rendered instead of the button boilerplate below.  This would provide a lot more flexibility for adding different action types: dropdowns, search/filter boxes, etc.